.github/workflows/test-and-lint.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       run: |
         pytest --cov-report xml --cov=fov_processing_pipeline fov_processing_pipeline/tests/
         codecov -t ${{ secrets.CODECOV_TOKEN }}
+      env:
+        CI: True
```

Comment:
You are adding a step to the workflow. Don't know yet what CI means. But setting it to True is one thing that you can do.

Reply:
"Continuous Integration": Automatically build and test your code as you push it to GitHub, preventing bugs from being deployed to production.